src/server/resolve.rs

```diff
@@ -133,3 +137,90 @@ impl StoreAddrResolver for MockStoreAddrResolver {
         unimplemented!();
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
can we mock a PD server in pd mod, so we don't need to start a real PD server in CI test.